package.json

```diff
@@ -66,6 +69,7 @@
     "prettier": "^2.8.8",
     "ts-node": "^10.9.1",
     "tsup": "^6.7.0",
-    "typescript": "^5.0.4"
+    "typescript": "^5.0.4",
+    "vitepress": "^1.1.0"
   }
-}
+}
```

Comment:
trailing newlineeeeee

Reply:
Added trailing newlineeeeee in 1289ab2e23a0b185962533363eaa925ef6f50cb1